compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/Focusable.kt

```diff
@@ -141,7 +141,8 @@ private val FocusableInNonTouchModeElement =
 
         override fun update(node: FocusableInNonTouchMode) {}
 
-        override fun hashCode(): Int = System.identityHashCode(this)
+        private val arbitraryHashCode: Int = 0x70f4de02 // Just a randomly generated constant
```

Comment:
Should we copy the value from java instead of generate a new one?

Reply:
js and native don't have an alternative for System.identityHashCode, so we can't really copy it.
And on every launch it can be different (in jvm).

I'm just not sure why they don't use the default hashCode here.

Reply:
Maybe copying from jvmMain is a valid option then.